flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/ParquetColumnarRowInputFormatTest.java

```diff
@@ -294,56 +286,6 @@ void testProjectionReadUnknownField(int rowGroupSize) throws IOException {
                 });
     }
 
-    @ParameterizedTest
-    @MethodSource("parameters")
-    void testPartitionValues(int rowGroupSize) throws IOException {
```

Comment:
After remove the invalid test in  parquet/orc input format test, I think we should add test for `FileInfoExtractorBulkFormat` to make sure it can get partition columns correctly.